Notation.Plugin.AzureKeyVault/Certificate/Pkcs12.cs

```diff
@@ -0,0 +1,39 @@
+using System.Security.Cryptography.Pkcs;
+using Notation.Plugin.Protocol;
+
+namespace Notation.Plugin.AzureKeyVault.Certificate
+{
+    static class Pkcs12
+    {
+        /// <summary>
+        /// Re-encode the PKCS12 data to removed the MAC.
+        /// The macOS doesn't support PKCS12 with non-encrypted MAC.
+        /// </summary>
+        /// <param name="data"></param>
+        /// <returns></returns>
+        /// <exception cref="ValidationException"></exception>
+        public static byte[] RemoveMac(byte[] data)
+        {
+            Pkcs12Info pfx = Pkcs12Info.Decode(data, out _);
+            // only remove the MAC if it is password protected
+            if (pfx.IntegrityMode != Pkcs12IntegrityMode.Password)
+            {
+                return data;
+            }
+            // verify the MAC with empty password
```

Comment:
```suggestion
            // verify the MAC with null password
```
Reference: https://learn.microsoft.com/en-us/dotnet/api/system.security.cryptography.pkcs.pkcs12info.verifymac?view=dotnet-plat-ext-7.0
> In the PKCS#12 specification, a distinction is made between a null password and an "empty" password, and that difference is reflected in the return value of this method. ReadOnlySpan<Char>.Empty represents the null password and String.Empty.AsSpan() represents the "empty" password. If one of those two values returns true, the other won't.

Reply:
Updated.